redash/handlers/groups.py

```diff
@@ -67,6 +67,10 @@ def delete(self, group_id):
         if group.type == models.Group.BUILTIN_GROUP:
             abort(400, message="Can't delete built-in groups.")
 
+        members = models.Group.members(group_id)
+        for member in members:
+            member.group_ids.remove(int(group_id))
```

Comment:
Need to add `member` to the session too?

Reply:
@arikfr It was my mistake. Fixed it. Thanks.